resources/services/accessanalyzer/analyzers_mock_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build mock
 // +build mock
```

Comment:
Should we remove old tags?

Reply:
I ran this gofmt command from the root of the repo and this is what it did  `gofmt -s -w .` ... gofmt behavior is usually taken as canonical

Reply:
I have opened an issue to add gofmt to our ci https://github.com/cloudquery/.github/issues/1